Alloy/alloy.js

```diff
@@ -53,7 +53,12 @@ program
 	.option('-y, --line <line>', 'Line for source map query', 1)
 	.option('-z, --source <source>', 'Source original file for source map query')
 	.option('--widgetname <name>', 'Widget name, used with generate command')
-	.option('--testapp <name>', 'Test app name to import, used with new command');
+	.option('--testapp <name>', 'Test app name to import, used with new command')
+	// PurgeTSS options
+	.option('--all', 'Run all processes: `purgetss build-fonts`, `purgetss build` and `purgetss`')
+	.option('--debug', 'Show time taken to execute each process.')
```

Comment:
I wonder if we should add a `purgetss` prefix here since it might be useful to use `--debug` for normal alloy commands at one point. I see people use it for plain alloy and not seeing any debug output (like --log-level debug).